src/InformaticsGateway/nlog.config

```diff
@@ -30,7 +30,7 @@
     <!--Console Target for hosting lifetime messages to improve Docker / Visual Studio startup detection -->
     <target xsi:type="ColoredConsole" name="lifetimeConsole" layout="${longdate}|${event-properties:item=EventId:whenEmpty=0}|${uppercase:${level}}|${logger}|${scopenested}|${message} ${exception:format=tostring}" />
 
-    <target name="logstash" xsi:type="Network" address="${environment:LOGSTASH_URL}" onOverflow="Discard">
+    <target name="logstash" xsi:type="Network" address="${environment:LOGSTASH_URL}" onOverflow="Discard" MaxMessageSize="64000">
```

Comment:
Just curious, any reason to use `Discard` instead of the default `Split`?

Reply:
yeah, your right, this was a leftover from something else, but I was logging large bodies, Ive simplified it now as I dont think it will be an issue.